airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -69,6 +76,12 @@ def __init__(
             validate_body_on_init {bool} -- If true, validates the yaml in the constructor,
                 setting this to True, will slow dag creation.
                 (default: {from env/airflow config: AIRFLOW__KUBE_JOB_OPERATOR__validate_body_on_init or False})
+            enable_jinja {bool} -- If true, the following fields will be parsed as jinja2,
+                        command, arguments, image, envs, body, namespace, config_file, cluster_context
+            resolve_relative_path_callstack_offset {int} - The call stack offset to render relative files
+                from.
+                    0 = relative to the caller of the constructor.
+                    1 = relative to the caller of the caller to the constructor.
```

Comment:
The documentation is unclear. How/why would you use this parameter?

Reply:
I see, this is a default parameter for class derivation. So, when
```
class MyCustomKubOperator(...):
       def __init__(..., resolve_relative_path_callstack_offset=1):
           super(resolve_relative_path_callstack_offset= resolve_relative_path_callstack_offset+1)
```
Otherwise relative input paths will not work. I can, in general, auto detect the number of past constructors called and use that to do auto path resolve. But for now this is the solution.

Reply:
Changed to class property.